### PR TITLE
Small fix

### PR DESCRIPTION
**Pull request by nickhsmith**
small fixes to MAE and to tests to match snakemake version update. snakemake DAG error in `no_run` test. so changed to run once until index, and then separately to finish solves it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The manuscript is available in [Nature Protocols](https://www.nature.com/article
 DROP is available on [bioconda](https://anaconda.org/bioconda/drop).
 We recommend using a dedicated conda environment. (installation time: ~ 10min)
 ```
-mamba install -c conda-forge -c bioconda drop
+mamba create -n drop_env -c conda-forge -c bioconda drop
```
Comment (vyepez88) [inline README.md:14]: why not explaining 2 scenarios. One where we already have a conda env and one where we do not.

File: drop/modules/mae-pipeline/MAE/ASEReadCounter.sh [hunk 2]
```diff
@@ -45,6 +45,20 @@ fi
 chr_subset=$(comm -12 <(cut -f1 -d" " ${canonical} | sort -u) <(echo "${vcf_chr}"))
 chr_subset=$(comm -12 <(echo "${bam_chr}") <(echo "${chr_subset}") | uniq)
 
+# ASEReadCounter fails without RG, this snippet checks for RG in bam file
```
Comment (vyepez88) [inline drop/modules/mae-pipeline/MAE/ASEReadCounter.sh]: add a comment saying that RG stands for read groups

File: README.md (hunk 1 above)
Comment (nickhsmith) [reply README.md:14]: Do we want to deal with an existing conda environment and the potential mismatches/headaches of that? Or ask people to create a drop specific one?

Comment (vyepez88) [reply README.md:14]: Why don't we give the 2 possibilites?
```
create conda env
activate it
install drop


install drop and create a conda env in one go
```

**Pull request merged**